### DISCUSSION TITLE
Is there a way to use a pinia store created by the main thread in a web-worker ?

### DISCUSSION THREAD

**Discussion by stillgbx** [vuejs/pinia#1308] [Help and Questions]
Please see this minimal test repo: https://github.com/stillgbx/pinia-web-worker-issue

**Comment by HoyosJuan** [comment]
> Please see this minimal test repo: https://github.com/stillgbx/pinia-web-worker-issue

Did you manage to be able to use the store from a worker?

**Reply by stillgbx** [reply]
Unfortunately no
